README.md

```diff
@@ -176,7 +181,7 @@ Given a WDL file, this runs the full syntax checker over the file and resolves i
 Error if a `call` references a task that doesn't exist:
 
 ```
-$ java -jar ../target/scala-2.11/cromwell-0.7.jar validate 2.wdl
+$ java -jar ../target/scala-2.11/cromwell.jar validate 2.wdl
```

Comment:
Really minor point, if this is so that the doc doesn't need updating each new version, I think this would be neater if it was just:
java -jar cromwell.jar validate 2.wdl

It would also make much more sense to someone who just has the jar and not the surrounding directory structure.